tests/seed_isort_config_test.py

```diff
@@ -63,7 +64,10 @@ def test_integration_editorconfig(tmpdir):
         assert tmpdir.join('.editorconfig').read() == expected
 
 
-@pytest.mark.parametrize('filename', ('setup.cfg', 'tox.ini'))
+@pytest.mark.parametrize(
+    'filename',
+    set(SUPPORTED_CONF_FILES) - {'.isort.cfg'},
+)
```

Comment:
heh yeah sorry I swapped this out from under you -- can rebase this chunk out (I ended up making a separate test for `.editorconfig`)

Reply:
Sure